### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Shevchuchka**
- [DEMO LINK](https://Shevchuchka.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Shevchuchka.github.io/layout_product-cards/report/html_report/)

- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,12 +2,53 @@
...
+        </section>
+        <p class="review__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="price">
+        <p class="price__text">Price:</p>
+        <h1 class="price__number">$2,199</h1>
+      </div>
+
+      <div class="buy">
```
Comment (etojeDenys) [inline src/index.html:44]: we make element a BEM block if we can reuse it somewhere, as an independent block

it seems to me that we will not be able to reuse these elements anywhere, so it is better to make them simple elements of the card block

you must to have only two blocks here: "card" and "stars"

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/styles/stars.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```
Comment (loralevitska) [inline src/styles/stars.scss:11]: `background` shorthand can be used here

**Review by loralevitska — CHANGES_REQUESTED** [review]
Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

File: src/index.html (hunk 1 above)
Comment (OleziO) [reply src/index.html:44]: Rename class according to the BEM

File: src/styles/card.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.card-block {
+  border: 1px #F3F3F3 solid;
+  border-radius: 5px;
+  box-sizing: border-box;
```
Comment (OleziO) [inline src/styles/card.scss:4]: Move to the * selector and remove it from all other places.

**Review by OleziO — CHANGES_REQUESTED** [review]

**Pull request closed without merging**